src/sigal/gallery.py

```diff
@@ -935,8 +936,23 @@ def remove_files(self, medias):
 
     def process_dir(self, album, force=False):
         """Process a list of images in a directory."""
+        def forcing(a):
+            if force is None:
+                return False
+            if isinstance(force, bool):
+                return force
+            elif len(force) == 0:
+                return True
```

Comment:
Same here, should not happen with the cli, could happen with the Python API but then I would just iterate on the list (with the block below).